lldb/include/lldb/Core/Mangled.h

```diff
@@ -44,7 +44,10 @@ class Mangled {
     eManglingSchemeMSVC,
     eManglingSchemeItanium,
     eManglingSchemeRustV0,
-    eManglingSchemeD
+    eManglingSchemeD,
```

Comment:
I'm curious, what's the mangling scheme that clang++? I don't see an obvious one.

Reply:
Itanium. All C++ compilers outside of MSCV implement the Itanium ABI.